### PR TITLE
fix rpm calculation

### LINKED ISSUE #53: rpm calculation is not correct
## Abstract

Motor does not rotate at correct speed even though `gear_ratio` and `num_motor_pole_pairs` are set correctly.

### PR DESCRIPTION
**Pull request by jsupratman13**
## Change Summary
- partially fix #53
- fix rpm calculation on velocity duty mode

## Additional Information

- will fix calculation on servo-side on next PR

### CODE REVIEW

File: vesc_hw_interface/src/vesc_hw_interface.cpp [hunk 1]
```diff
@@ -286,11 +290,11 @@ void VescHwInterface::packetCallback(const std::shared_ptr<VescPacket const>& pa
     std::shared_ptr<VescPacketValues const> values = std::dynamic_pointer_cast<VescPacketValues const>(packet);
 
     const double current = values->getMotorCurrent();
-    const double velocity_rpm = values->getVelocityERPM() / static_cast<double>(num_motor_pole_pairs_);
+    const double velocity_rpm = values->getVelocityERPM() / static_cast<double>(num_rotor_pole_pairs_);
     const double position_pulse = values->getPosition();
```
Comment (Tacha-S) [inline vesc_hw_interface/src/vesc_hw_interface.cpp:302]:
```suggestion
    const double position_steps = values->getPosition();
```

Comment (Tacha-S) [reply vesc_hw_interface/src/vesc_hw_interface.cpp:302]: It was not code for `wheel controller`, so you can ignore it.

File: vesc_hw_interface/README.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ All of following parameters are in `${VESC_HW_INTERFACE_NODE_NAME}/` namespace.
 - `port` (string, **required**): port name connecting to your VESC, *e.g.* `/dev/ttyUSB0`.
 - `command_mode` (string, **required**): control mode you want to use. Enter one of following parameters: `position`, `velocity`, `effort` and `effort_duty`.
 - `joint_name` (string, *default*: `joint_vesc`): corresponding joint name in your robot URDF.
-- `num_motor_pole_pairs` (double, *default*: 1.0): the number of motor pole pairs.
+- `num_rotor_pole_pairs` (double, *default*: 1.0): the number of rotor pole pairs.
```
Comment (nyxrobotics) [inline vesc_hw_interface/README.md]: 内部の計算に合わせるため、パラメータをnum_rotor_polesに変更しましょう

Comment (jsupratman13) [reply vesc_hw_interface/README.md]: sure

**Review by nyxrobotics — APPROVED** [review]

**Pull request merged**